src/libs/antares/study/binding_constraint/BindingConstraint.cpp

```diff
@@ -1537,10 +1530,75 @@ BindingConstraint* BindingConstraintsList::add(const AnyString& name)
     return bc;
 }
 
+void BindingConstraintsList::resizeAllTimeseriesNumbers(unsigned int nb_years) {
+    std::set<std::string> group_names;
+    std::for_each(pList.begin(), pList.end(), [&group_names](const auto& bc) {
+        group_names.insert(bc->group());
+    });
+    std::for_each(group_names.begin(), group_names.end(), [&](const auto& group_name) {
+        time_series[group_name].timeseriesNumbers.clear();
+        time_series[group_name].timeseriesNumbers.resize(1, nb_years);
+    });
+}
+
+bool BindingConstraint::loadTimeSeries(BindingConstraint::EnvForLoading &env) {
+    return loadBoundedTimeSeries(env, operatorType());
+}
+
+bool
+BindingConstraint::loadBoundedTimeSeries(BindingConstraint::EnvForLoading &env, BindingConstraint::Operator operatorType) {
```

Comment:
Renaming :  
**loadBoundedTimeSeries(...)** : we gave up the **bound** thing in the TS attribute name (**time_series**).
Should be renamed into **loadRHStimeSeries(...)**

Besides shouldn't this method be **private** ?